backend/gn_module_monitoring/monitoring/models.py

```diff
@@ -639,28 +643,32 @@ class TMonitoringModules(TModules, PermissionModel):
     cascade="all",
     lazy="select",
     uselist=False,
+    overlaps="sites",
 )
 
 TMonitoringSitesGroups.visits = DB.relationship(
     TMonitoringVisits,
     primaryjoin=(TMonitoringSites.id_sites_group == TMonitoringSitesGroups.id_sites_group),
     secondaryjoin=(TMonitoringVisits.id_base_site == TMonitoringSites.id_base_site),
     secondary="gn_monitoring.t_site_complements",
+    overlaps="sites,sites_group",
 )
 
 TMonitoringSitesGroups.nb_visits = column_property(
-    select([func.count(TMonitoringVisits.id_base_site)]).where(
+    select(func.count(TMonitoringVisits.id_base_site))
+    .where(
         and_(
```

Comment:
```suggestion
        
```
L'opérateur AND est ajouté par défaut entre les `whereclause` indiquées dans le `where()`